EduNLP/SIF/sif.py

```diff
@@ -59,14 +63,12 @@ def to_sif(item):
     >>> siftext
     '某校一个课外学习小组为研究某作物的发芽率$y$和温度$x$（单位...'
     """
-    item_parser = Parser(item)
-    item_parser.description_list()
-    item = item_parser.text
-    return item
+    _, sif_item = is_sif(item, check_formula)
+    return sif_item
 
 
 def sif4sci(item: str, figures: (dict, bool) = None, safe=True, symbol: str = None, tokenization=True,
```

Comment:
When you sepecify some modes such as `ast`  in tokenzation_params, is it possible to set check_formula as `False`?

Reply:
It is ok, but tokenzation_params mainly work during 'tokenize', and check_formula is used in is_sif before 'tokenize'. Maybe it is better to use seperately for clear direction. 
Or we can use it within 'safe':
 when safe = 0,  don't use is_sif and don't check anything
 when safe = 1,  don't use is_sif but don't check formula
 when safe = 2,  use is_sif and check formula

Reply:
`safe` looks like a boolean variable, maybe safe_mode is a better choice?

Reply:
Maybe we can also set a wise `safe` mode?